spec/yelp/error_spec.rb

```diff
@@ -30,15 +30,25 @@
     end
 
     it 'should expose the field parameter' do 
-      begin
+      expect{
         Yelp::Error.check_for_error(bad_response)
-      rescue Yelp::Error::InvalidParameter => e 
-        # verifies that StandardError message attribute is available
-        expect(e.message).to eq('One or more parameters are invalid in request: oauth_token')
-        # verifies that we can get access to the specific field that was invalid
-        expect(e.field).to eq('oauth_token')
-      end        
+      }.to raise_error { |error|
```

Comment:
`expect { ... }.to raise_error(Yelp::Error::InvalidParameter, 'One or more ...')`

[documentation](https://relishapp.com/rspec/rspec-expectations/v/2-5/docs/built-in-matchers/raise-error-matcher#expect-specific-error-message-using-a-string)